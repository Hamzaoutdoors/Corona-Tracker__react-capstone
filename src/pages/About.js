import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faAngleDoubleRight } from '@fortawesome/free-solid-svg-icons';
import { Link } from 'react-router-dom';
import AboutComponent from '../components/AboutCompo';

const About = () => (
  <div className="d-flex flex-column">
    <div className="page-title">
      <p data-content="Coronavirus disease (COVID-19) pandemic">Coronavirus disease (COVID-19) pandemic</p>
    </div>
    <Link to="/" className="arrow my-2 ms-auto p-2">
      {' '}
      Go Back Home
      <FontAwesomeIcon icon={faAngleDoubleRight} className="mx-1" />
    </Link>
    {/* Counties list */}
    <AboutComponent />
  </div>
);

export default About;
