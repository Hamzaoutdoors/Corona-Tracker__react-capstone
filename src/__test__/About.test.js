import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import About from '../pages/About';

test('About page renders properly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <About />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
