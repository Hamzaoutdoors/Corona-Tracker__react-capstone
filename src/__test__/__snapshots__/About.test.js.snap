// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders properly 1`] = `
<div
  className="d-flex flex-column"
>
  <div
    className="page-title"
  >
    <p
      data-content="Coronavirus disease (COVID-19) pandemic"
    >
      Coronavirus disease (COVID-19) pandemic
    </p>
  </div>
  <a
    className="arrow my-2 ms-auto p-2"
    href="/"
    onClick={[Function]}
  >
     
    Go Back Home
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-angle-double-right fa-w-14 mx-1"
      data-icon="angle-double-right"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34zm192-34l-136-136c-9.4-9.4-24.6-9.4-33.9 0l-22.6 22.6c-9.4 9.4-9.4 24.6 0 33.9l96.4 96.4-96.4 96.4c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l136-136c9.4-9.2 9.4-24.4 0-33.8z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </a>
  <div>
     
    <div
      className="page-header"
    >
      <div
        className="header-image"
      >
        <img
          alt="cart"
          src="/cartes/globe.svg"
        />
      </div>
      <div
        className="header-body"
      >
        <h2>
          About US
        </h2>
        <p>
          
        </p>
      </div>
    </div>
    <div
      className="countries-container container"
    >
      <div>
        <h1
          className="my-4 shadow"
        >
          About COVID19 API
        </h1>
        <p>
          The full API dataset is updated each day between
           
          <strong>
             1PM 
          </strong>
           
          and
          <strong>
             3PM 
          </strong>
           
          CET with additional updates throughout the day.
        </p>
        <p>
          Data is provided by the following institutions:
        </p>
        <ul>
          <li>
            <a
              href="https://systems.jhu.edu/research/public-health/ncov/"
            >
              Johns Hopkins University
            </a>
          </li>
          <li>
            <a
              href="https://github.com/pcm-dpc/COVID-19"
            >
              Dipartimento della Protezione Civile of Italy
            </a>
          </li>
          <li>
            <a
              href="https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/Gesamt.html"
            >
              Robert Koch Institute of Germany
            </a>
          </li>
          <li>
            <a
              href="https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov/situacionActual.htm"
            >
              Ministry of Health Spain
            </a>
          </li>
          <li>
            <a
              href="https://www.santepubliquefrance.fr/dossiers/coronavirus-covid-19"
            >
              Santé publique France
            </a>
          </li>
        </ul>
        <p
          className="text-monospace warning"
        >
          All Copyright © and Thanks For
          <a
            className="text-decoration-none"
            href="https://covid19tracking.narrativa.com/index_en.html"
          >
             Narrativa API
          </a>
        </p>
      </div>
      <div>
        <h1
          className="my-4 shadow"
        >
          About Constructer
        </h1>
        <p>
          Hamza Ellaouzi Full-Stack Developer, Geoinformatics Engineer and Lifelong Learner. Based in Tangier, Morocco Building stuff with new technlogies become my daily Routine
        </p>
        <p>
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-link fa-w-16 mx-2"
            data-icon="link"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          You can follow me on :
        </p>
        <ul
          className="d-flex justify-content-between "
        >
          <li
            className="badge badge-pill badge-primary "
          >
            <a
              className="text-decoration-none text-dark"
              href="https://github.com/Hamzaoutdoors"
              rel="noreferrer"
              target="_blank"
            >
              Github
            </a>
          </li>
          <li
            className="badge badge-pill badge-primary"
          >
            <a
              className="text-decoration-none text-dark"
              href="https://web.facebook.com/?_rdc=1&_rdr"
              rel="noreferrer"
              target="_blank"
            >
              Facebook
            </a>
          </li>
          <li
            className="badge badge-pill badge-primary"
          >
            <a
              className="text-decoration-none text-dark"
              href="https://www.linkedin.com/in/hamzaellaouzi/?locale=en_US"
              rel="noreferrer"
              target="_blank"
            >
              LinkedIn
            </a>
          </li>
        </ul>
        <p
          className="text-monospace warning"
        >
          All Copyright © For
          <a
            className="text-decoration-none"
            href="https://covid19tracking.narrativa.com/index_en.html"
          >
             Narrativa API
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;
